azext_edge/edge/_help.py

```diff
@@ -412,10 +412,45 @@ def load_iotops_help():
 
         - name: Add a data point to an asset with the given pre-filled values.
           text: >
-            az iot ops asset data-point add --asset MyAsset -g MyRG --data-source nodeId1 --name tagName1
+            az iot ops asset data-point add --asset MyAsset -g MyRG --data-source NodeID1 --name tagName1
             --capability-id tagId1 --observability-mode log --queue-size 5 --sampling-interval 200
     """
 
+    helps[
+        "iot ops asset data-point export"
+    ] = """
+        type: command
+        short-summary: Export data points in an asset.
+        long-summary: The file name will be {asset_name}_dataPoints.{file_type}.
+        examples:
+        - name: Export all data points in an asset in JSON format.
+          text: >
+            az iot ops asset data-point export --asset {asset} -g {resource_group}
+        - name: Export all data points in an asset in CSV format in a specific output directory.
+          text: >
+            az iot ops asset data-point export --asset {asset} -g {resource_group} --format csv --output-dir {output_directory}
+        - name: Export all data points in an asset in CSV format that can be uploaded via the DOE portal.
+          text: >
+            az iot ops asset data-point export --asset {asset} -g {resource_group} --format portal-csv
+        - name: Export all data points in an asset in YAML format. Replace the file if one is present already.
+          text: >
+            az iot ops asset data-point export --asset {asset} -g {resource_group} --format yaml --replace
+    """
+
+    helps[
+        "iot ops asset data-point import"
+    ] = """
+        type: command
+        short-summary: Import data points in an asset.
```

Comment:
Given the current way file extensions are used for processing, we can indicate in the [long-]summary something like "import supports specifically these file extension formats..."